refactor: update init script

Thienhuynh95 opened the pull request:
## Description
- update zonosApiKey
- update condition to only add zonos script when `/loadZonos.js` doesn't exist

**Note**: To make sure it works correctly on your own computer, the development server must use the HTTPS protocol. Here's what you should see when you run it on your computer: 
- Checkout modal shows up after clicking the `Delivery Page` button:
  <img width="1259" alt="image" src="https://github.com/chenhanting-ibuypower/hyte-international-affair/assets/17950626/827a2017-29c6-42de-8e06-4dddb8bb2b77">

Thienhuynh95 committed:
refactor: update init script
- update zonosApiKey
- update condition to only add zonos script when `/loadZonos.js` doesn't
exist

chenhanting-ibuypower commented:
> ## Description
> * update zonosApiKey
> * update condition to only add zonos script when `/loadZonos.js` doesn't exist
> 
> **Note**: To make sure it works correctly on your own computer, the development server must use the HTTPS protocol. Here's what you should see when you run it on your computer:
> 
> * Checkout modal shows up after clicking the `Delivery Page` button:
>   <img alt="image" width="1259" src="https://private-user-images.githubusercontent.com/17950626/299826872-827a2017-29c6-42de-8e06-4dddb8bb2b77.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYyMzUxNjcsIm5iZiI6MTcwNjIzNDg2NywicGF0aCI6Ii8xNzk1MDYyNi8yOTk4MjY4NzItODI3YTIwMTctMjljNi00MmRlLThlMDYtNGRkZGI4YmIyYjc3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI2VDAyMDc0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE4NTFkODA3MzQzZGQ5NTBhNjE2YjJiNzlkNzRiNzFiNTUwZWI0NTZjNDVkOWUyOTdiYzAzMzAyYWQ3YTYyMzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.10bvtxypda7lo-ykXkYRWBpvr389AoIJJDh4Y7ycQGA">

==============

update condition to only add zonos script when `/loadZonos.js` doesn't exist

==============

Can you tell me how it recognize `/loadZonos.js`. I think it doesn't work, it only runs 
`https://js.zonos.com/dist/scripts/loadZonos.js?timestamp=${timestamp}`, isn't it?

But fortunately it works. Thank you

chenhanting-ibuypower commented:
Hey Thien, @Thienhuynh95 

It seems like it does actual payment 
<img width="906" alt="image" src="https://github.com/chenhanting-ibuypower/hyte-international-affair/assets/105334528/2dcc6780-8774-49a5-b360-d55f7b2c1bde">

Thienhuynh95 commented:
> > ## Description
> > 
> > * update zonosApiKey
> > * update condition to only add zonos script when `/loadZonos.js` doesn't exist
> > 
> > **Note**: To make sure it works correctly on your own computer, the development server must use the HTTPS protocol. Here's what you should see when you run it on your computer:
> > 
> > * Checkout modal shows up after clicking the `Delivery Page` button:
> >   <img alt="image" width="1259" src="https://private-user-images.githubusercontent.com/17950626/299826872-827a2017-29c6-42de-8e06-4dddb8bb2b77.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYyMzUxNjcsIm5iZiI6MTcwNjIzNDg2NywicGF0aCI6Ii8xNzk1MDYyNi8yOTk4MjY4NzItODI3YTIwMTctMjljNi00MmRlLThlMDYtNGRkZGI4YmIyYjc3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI2VDAyMDc0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE4NTFkODA3MzQzZGQ5NTBhNjE2YjJiNzlkNzRiNzFiNTUwZWI0NTZjNDVkOWUyOTdiYzAzMzAyYWQ3YTYyMzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.10bvtxypda7lo-ykXkYRWBpvr389AoIJJDh4Y7ycQGA">
> 
> ==============
> 
> update condition to only add zonos script when `/loadZonos.js` doesn't exist
> 
> ==============
> 
> Can you tell me how it recognize `/loadZonos.js`. I think it doesn't work, it only runs `https://js.zonos.com/dist/scripts/loadZonos.js?timestamp=${timestamp}`, isn't it?
> 
> But fortunately it works. Thank you

Hi Han, I may not have completely understood your question, so there's a chance my answer might not directly address your concerns. The condition I implemented is designed to add the `/loadZonos.js` script to the DOM only if the script with the URL `js.zonos.com` is not already there. I've also updated the URL from `js.zonos.com` to `alpha.js.zonos.com`. This change is meant to enable faster updates and support from my side until the Hyte integration is stable and fully functional. Once everything is working smoothly, we'll launch the new version. At that point, you can choose to use the most recent version at js.zonos.com or stick with a specific version (ex: v1.2.4.js.zonos.com).

Thienhuynh95 commented:
> Hey Thien, @Thienhuynh95
> 
> It seems like it does actual payment <img alt="image" width="906" src="https://private-user-images.githubusercontent.com/105334528/299869222-2dcc6780-8774-49a5-b360-d55f7b2c1bde.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYyODU5MTgsIm5iZiI6MTcwNjI4NTYxOCwicGF0aCI6Ii8xMDUzMzQ1MjgvMjk5ODY5MjIyLTJkY2M2NzgwLTg3NzQtNDlhNS1iMzYwLWQ1NWY3YjJjMWJkZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyNlQxNjEzMzhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jYjY2NmMxMWQ1NDUxZmE3ZjZiMGU4Mzk3OGQxYjMyYjUxNWM5ZGE1YjRmMGE2ZjdiZDQ5MTUwYmY2ZDcwOTQyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.bO2e3OjKMvnZfICDjlSbbzep9mjP0O5YNY9WQd6dk2A">

Yes, the last payment step would make an actual payment.
